test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -29,7 +29,7 @@ public class AuditTests : IClassFixture<AuditTestFixture>
     {
         private const string RequestIdHeaderName = "X-Request-Id";
         private const string CustomAuditHeaderPrefix = "X-MS-AZUREFHIR-AUDIT-";
-        private const string ExpectedClaimKey = "client_id";
+        private const string ExpectedClaimKey = "appid";
```

Comment:
Is this an intentional change? I ask because the change was made address an issue with running the tests locally. (Maybe just an old commit without merging master.)

Reply:
It was not intentional. Reverted it.